audio-to-text/README.md

```diff
@@ -0,0 +1,105 @@
+# Audio to Text Application
+
+  This sample application is a simple recipe to transcribe an audio file.
+  This provides a simple recipe to help developers start building out their own custom LLM enabled
+  audio-to-text applications. It consists of two main components; the Model Service and the AI Application.
+
+  There are a few options today for local Model Serving, but this recipe will use [`whisper-cpp`](https://github.com/ggerganov/whisper.cpp.git)
+  and their OpenAI compatible Model Service. There is a Containerfile provided that can be used to build this Model Service within the repo,
```

Comment:
I don't think this model uses an OpenAI compatible API.
https://platform.openai.com/docs/api-reference/audio/createTranscription?lang=curl

We can probably just say, "and their included model server"

Reply:
done